src/main/java/edu/hawaii/its/api/service/GroupingsService.java

```diff
@@ -25,15 +31,50 @@
 @Service
 public class GroupingsService {
 
+    @Value("${groupings.api.grouping_admins}")
+    private String GROUPING_ADMINS;
+
     @Value("${groupings.api.trio}")
     private String TRIO;
 
+    public static final Log logger = LogFactory.getLog(GroupingAssignmentService.class);
+
     @Autowired
     private GroupPathService groupPathService;
 
     @Autowired
     private GrouperApiService grouperApiService;
 
+    @Autowired
+    private MemberService memberService;
+
+    @Autowired
+    private GroupingAssignmentService groupingAssignmentService;
+
+    /**
+     * A list of grouping paths for all groupings, restricted to admins' use only.
+     */
+    public GroupingPaths groupingPaths(String adminUhIdentifier) {
+        logger.info(String.format("allGroupings; adminUhIdentifier: %s;", adminUhIdentifier));
+        if (!memberService.isAdmin(adminUhIdentifier)) {
+            throw new AccessDeniedException();
+        }
+        return new GroupingPaths(this.allGroupingPaths());
+    }
+
+    /**
+     * Returns groupingsAdmins object containing the list of all admins.
+     */
+    public GroupingAdmins groupingAdmins (String adminUhIdentifier) {
+        logger.info(String.format("groupingAdmins; adminUhIdentifier: %s;", adminUhIdentifier));
+        if (!memberService.isAdmin(adminUhIdentifier)) {
+            throw new AccessDeniedException();
+        }
+        List<String> adminGrouping = Arrays.asList(GROUPING_ADMINS);
+        edu.hawaii.its.api.type.Group admin = groupingAssignmentService.getMembers(adminUhIdentifier, adminGrouping).get(GROUPING_ADMINS);
```

Comment:
I would also put `Arrays.asList(GROUPING_ADMINS)` directly into the `getMembers()` function to avoid any confusion with setting it to the variable a`dminGrouping`. Thanks.